src/utils/buffer.ts

```diff
@@ -81,7 +80,10 @@ export const unpackToInt = (n: number, signed: boolean, buff: Buffer): [number,
     }
   } else if (n === 8) {
     // We have only signed ints here
-    if (parseInt(nodeMajorVersion) > 8) {
+    const nodeMajorVersion = get(process, 'versions.node');
```

Comment:
Will this return undefined in browser? You can use a default value as 3rd arg or use `_.has` to check if this property exists

Reply:
This code has changed, can you see again please?